lib/utilities/monitoringHandler.js

```diff
@@ -64,6 +64,31 @@ const httpResponseSizeBytes = new client.Summary({
     help: 'Cloudserver HTTP response sizes in bytes',
 });
 
+const quotaEvaluationDuration = new client.Histogram({
+    name: 's3_cloudserver_quota_evaluation_duration_seconds',
+    help: 'Duration of the quota evaluation operation',
+    labelNames: ['action', 'code', 'type'],
+    buckets: [0.0001, 0.005, 0.015, 0.05, 0.1, 0.2, 0.3, 0.4, 0.5],
+});
+
+const utilizationMetricsRetrievalDuration = new client.Histogram({
+    name: 's3_cloudserver_utilization_metrics_retrieval_duration_seconds',
+    help: 'Duration of the utilization metrics retrieval operation',
+    labelNames: [...labelNames, 'class'],
+    buckets: [0.0001, 0.00025, 0.005, 0.015, 0.05, 0.1, 0.2, 0.3, 0.4, 0.5],
+});
+
+const utilizationServiceHealth = new client.Counter({
+    name: 's3_cloudserver_utilization_service_state',
```

Comment:
why not call it scuba ?

Reply:
I don't know if we want to use Scuba publicly, because it was supposed to be dropped in favor of SUR, but anyway the reason is that I wanted to dissociate the naming of the metric with the backend, as the backend can change independently of cloudserver...

Reply:
`utilization_service_state` seems out of place (the state of the utilization service should be monitored in that service... here we monitor the availability of the feature in cloudserver which depends on it, i.e. the impact of unavailability of that service?), and it feels like we loose the "relationship" with the quota.

maybe something like `s3_cloudserver_quota_availaibility` or `s3_cloudserver_quota_state` ?

Reply:
Done in https://github.com/scality/cloudserver/pull/5556/files#diff-3daa33d35f5375a7070d74a0b174c01ab8b982392db4025ebc49cf3690aa16bcR91

I still have the utilization service inside as this is what defines the availability of the feature